vgtk/src/lib.rs

```diff
@@ -633,10 +633,15 @@ pub fn start<C: 'static + Component>() -> (Application, Scope<C>) {
         });
     });
 
-    app.connect_activate(move |_| {
+    let id_cell = std::rc::Rc::new(std::cell::RefCell::new(None));
+    let id_cell_clone = id_cell.clone();
+    id_cell.replace(Some(app.connect_activate(move |app| {
         debug!("{}", "Application has activated.".bright_blue());
         constructor(());
-    });
+        if let Some(id) = id_cell_clone.replace(None) {
+            glib::signal::signal_handler_disconnect(app, id);
+        }
+    })));
 
```

Comment:
What is the purpose of this code?

Reply:
This was my fix for the same issue as https://github.com/bodil/vgtk/commit/c6325f8bc43eafd98a32705204e15053a62eba60 addressed, but is (presumably) no longer necessary. I'm updating my fork now.